<launch>
	
	<!-- Start the DJI SDK -->
	<include file="$(find dji_sdk)/launch/sdkM.launch"/>

	<!-- Start Mavros -->
	<include file="$(find jetyak_uav_utils)/launch/jetyak_mavros.launch"/>

	<!-- Vision node -->
	<group ns="jetyak_uav_vision">

		<!-- Load vision and gimbal parameters -->
		<rosparam command="load" file="$(find jetyak_uav_utils)/cfg/gimbal_cam.yaml"/>

		<!-- Start the camera and gimbal controller -->
		<include file="$(find dji_gimbal_cam)/launch/default.launch"/>
		<node name="gimbal_tag" pkg="jetyak_uav_utils" type="gimbal_tag_node" output="screen" />

		<!-- Start tag tracking -->
		<include file="$(find jetyak_uav_utils)/launch/ar_track.launch"/>

<node pkg="vrpn_client_ros" type="vrpn_client_node" name="vrpn_client_node" output="screen">
<rosparam subst_value="true">
server: 192.168.1.102
port: 3883
update_frequency: 30.0
frame_id: world
use_server_time: false
broadcast_tf: true
trackers:
- Matrice
- MasterTag
</rosparam>
</node>


		<!-- Start colocalization filter -->
		<node name="filter" pkg="jetyak_uav_utils" type="optiStatePub.py" output="screen"/>
	</group>
</launch>
